rdkit/Chem/SimpleEnum/Enumerator.py

```diff
@@ -214,21 +216,17 @@ def _uniqueOnly(lst):
         if smi not in seen:
           seen.append(smi)
           yield entry
-
-  ps = AllChem.EnumerateLibraryFromReaction(reaction, bbLists)
-  if not uniqueProductsOnly:
-    return ps
-  else:
-    return _uniqueOnly(ps)
+  
+  return _uniqueOnly(ps)
```

Comment:
Not sure if this change adds anything.

Reply:
The old required the function `_uniqueOnly()` must be called regardless of consequences. This ensure we don't mess thing with. I will upgrade it